### DISCUSSION TITLE
Notification of force-push

### DISCUSSION THREAD

**Discussion by adamrankin** [PlusToolkit/PlusLib#961] [General]
Due to a misunderstanding in a research agreement, I have to eliminate commits from the history. Any affected files have not been modified since so a rebase onto the new history will be straightforward.

My sincere apologies for any complications that arise.

Adam

**Comment by lassoan** [comment]
Which files are impacted?

**Reply by adamrankin** [reply]
PlusBuild/CMakeLists.txt
PlusLib/CMakeLists.txt
PlusLib/src/CMakeLists.txt
PlusLib/src/PlusServer/CMakeLists.txt

Edit:
PlusLib/src/UsePlusLib.cmake.in
